xarray/backends/scipy_.py

```diff
@@ -188,8 +188,9 @@ def encode_variable(self, variable):
     def prepare_variable(self, name, variable, check_encoding=False,
                          unlimited_dims=None):
         if check_encoding and variable.encoding:
-            raise ValueError('unexpected encoding for scipy backend: %r'
-                             % list(variable.encoding))
+            if variable.encoding != {'_FillValue': None}:
```

Comment:
Yeah, this is a little awkward.

I think this is probably OK (backend classes already need to know about encodings), but it would be good to think about why we don't already have validation for encodings with scipy. We should probably be raising an error if somebody attempts an encoding like `{'zlib': True}` but right now it looks like we don't do that.